modules/javafx.controls/src/main/java/javafx/scene/control/skin/DatePickerSkin.java

```diff
@@ -142,6 +142,7 @@ public DatePickerSkin(final DatePicker control) {
                 }
                 show();
             } else {
+                updateDisplayNode();
```

Comment:
Can the call to `updateDisplayNode();` be removed in line 128 then?

Reply:
I believe `updateDisplayNode()` in line 128 is required as it is the event handler on `valueProperty`.

Reply:
I think so too.